packages/beacon-node/tsconfig.json

```diff
@@ -3,6 +3,7 @@
   "exclude": ["../../node_modules/it-pipe"],
   "include": ["src", "test"],
   "compilerOptions": {
-    "outDir": "lib"
+    "module": "node16",
+    "moduleResolution": "node16"
```

Comment:
Can we move `module` and `moduleResolution` to root tsconfig?
Would possibly like to use the same pattern with other packages in the monorepo

Reply:
We can't because some or the packages we have are also build for browsers and would like to keep them to `esm` specific modules. 

There would only be `beacon-node` and `cli` packages where we may need `node16` module resolution to swap some dependencies. And these are only 2, so I kept this tsconfig change only here for now.

Reply:
We may want to do the same thing with utils.
How does setting module and moduleResolution stop a module from being used in the browser?